atlp-rwanda/e-comm-team-techsmith-fn
# #185218918: two factor authentication for the seller

## What does this PR do?
sets up two factor authentication for seller user

## Description of Task to be completed?

For a seller to login, they must be authenticated with 2FA in order to ensure the security.

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run dev`

- Check the application's documentation on Storybook: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
- [#185218918]
